<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>mainGUI</class>
 <widget class="QMainWindow" name="mainGUI">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>640</width>
    <height>480</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>640</width>
    <height>480</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>640</width>
    <height>480</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>NetFlow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="verticalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>-1</x>
      <y>-1</y>
      <width>641</width>
      <height>481</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout" stretch="1,6,0">
       <item>
        <widget class="QLabel" name="label">
         <property name="text">
          <string>tshark路径</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="tsharkPath">
         <property name="text">
          <string>./WiresharkPortable64/App/Wireshark/tshark.exe</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="browseTshark">
         <property name="text">
          <string>浏览</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_5" stretch="1,6,0">
       <item>
        <widget class="QLabel" name="label_3">
         <property name="text">
          <string>文件保存位置</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="savePath">
         <property name="text">
          <string>./</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="browseSave">
         <property name="text">
          <string>浏览</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_2" stretch="1,6,0">
       <item>
        <widget class="QLabel" name="label_5">
         <property name="text">
          <string>模型路径</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="modelPath">
         <property name="text">
          <string>./model_0.9904_20241209_014305.joblib</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="browseModel">
         <property name="text">
          <string>浏览</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_4" stretch="0,1,0,1,0">
       <item>
        <widget class="QLabel" name="label_2">
         <property name="text">
          <string>抓包保存时间（秒）</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QSpinBox" name="timeInput">
         <property name="maximum">
          <number>86400</number>
         </property>
         <property name="singleStep">
          <number>1</number>
         </property>
         <property name="value">
          <number>10</number>
         </property>
         <property name="displayIntegerBase">
          <number>10</number>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_4">
         <property name="text">
          <string>网卡</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="interfaceCombo"/>
       </item>
       <item>
        <widget class="QPushButton" name="controlButton">
         <property name="text">
          <string>开始监控</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QTextBrowser" name="textOutput"/>
     </item>
    </layout>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
